### PR TITLE
Project setup and common modules

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: common.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  source                       = "git::https://github.com/nationalarchives/da-terraform-modules//vpc"
+  vpc_name                     = "${local.environment}-vpc"
+  az_count                     = local.az_count
+  elastic_ip_allocation_ids    = [aws_eip.eip.allocation_id]
+  nat_instance_security_groups = [module.nat_instance_security_group.security_group_id]
+}
+
+resource "aws_eip" "eip" {
+  domain = "vpc"
+}
```
Comment (kurtismash) [inline common.tf:38]: DOes 1 EIP allow us to be HA?

**Review by kurtismash — COMMENTED** [review]
A couple of comments.

File: common.tf (hunk 1 above)
Comment (MancunianSam) [reply common.tf:38]: Good point, I've put a count in with the az_count.

File: templates/secrets_manager/secrets_manager_lambda_policy.json.tpl
[the remark this answers is not in the document]
Comment (MancunianSam) [reply templates/secrets_manager/secrets_manager_lambda_policy.json.tpl:1]: I thought you needed a secrets manager policy to allow services to access it but I don't think you do so I've deleted this.

**Review by kurtismash — APPROVED** [review]

**Pull request merged**